edoweb/edoweb.module

```diff
@@ -2483,14 +2504,46 @@ function _edoweb_format_date($timestamp) {
   return format_date($timestamp, 'custom', 'D, d.m.Y - H:i:s');
 }
 
+function _edoweb_format_access_icons($entity) {
+  
+  $icons = '';
+  switch ($entity->access_data) {
+    case 'private':
+      $icons .= '<i class="fa fa-lock" title="' . t($entity->access_data) . '" style="color:red;"></i>';
+      break;
+    case 'subscriber':
+    case 'restricted':
+      $icons .= '<i class="fa fa-lock" title="' . t($entity->access_data) . '"style="color:orange;"></i>';
+      break;
+    case 'remote':
+      $icons .= '<i class="fa fa-lock" title="' . t($entity->access_data) . '"style="color:yellow;"></i>';
+      break;
+    case 'public':
+      $icons .= '<i class="fa fa-unlock" title="' . t($entity->access_data) . '"style="color:green;"></i>';
+      break;    
+  }
+ 
+  switch ($entity->access_md) {
+    case 'private':
+      $icons .= '&nbsp;&nbsp;&nbsp;&nbsp;<i class="fa fa-circle-o" title="' . t($entity->access_md) . '"style="color:red;"></i>';
+      break;
+    case 'public':
+      $icons .= '&nbsp;&nbsp;&nbsp;&nbsp;<i class="fa fa-circle-o" title="' . t($entity->access_md) . '"style="color:green;"></i>';
+      break;
+  }
+  
+  return $icons;
+
+}
+
 function _edoweb_format_subject($remote_id) {
   $curie = _edoweb_compact_uri($remote_id);
   return strtoupper(substr($curie, 0, strpos($curie, ':')));
 }
 
-function _edoweb_link_lastmodified($remote_id) {
-  if (_is_edoweb_entity($remote_id)) {
-    $curie = _edoweb_compact_uri($remote_id);
+function _edoweb_link_lastmodified($entity) {
+  if (('journal' == $entity->bundle()) && _is_edoweb_entity($entity)) {
```

Comment:
I already had this for all resources types and then saw somewhere the original requirement was to only display this for journals. Will change this!